manifests/traffics/shiny.yaml

```diff
@@ -86,9 +86,29 @@ spec:
     spec:
       containers:
       - name: shiny
-        image: codeformuenster/traffic-dynamics-shiny:v0.5.1.3
+        image: codeformuenster/traffic-dynamics-shiny:v0.6.0
         ports:
         - containerPort: 3838
         # FIXME
         # resources:
         #   null
+    volumeMounts:
```

Comment:
This volumeMounts needs more indentation. It has to be on the same level as `ports`